src/Repository/Doctrine/BlockRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Adshares\AdsOperator\Repository\Doctrine;
+
+use Adshares\AdsOperator\Repository\BlockRepositoryInterface;
+use Adshares\AdsOperator\Document\Block;
+
+/**
+ * Class BlockRepository
+ * @package Adshares\AdsOperator\Repository\Doctrine
+ */
+class BlockRepository extends BaseRepository implements BlockRepositoryInterface
+{
+    /**
+     * @return array
+     */
+    public function availableSortingFields(): array
+    {
+        return [
+            'id',
+            'time',
+        ];
+    }
+
+    /**
+     * @param string $blockId
+     * @return Block|null
+     * @throws \Doctrine\ODM\MongoDB\LockException
+     * @throws \Doctrine\ODM\MongoDB\Mapping\MappingException
+     */
+    public function getBlock(string $blockId):? Block
+    {
+        /** @var Block $block */
+        $block = $this->find($blockId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$block". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)